slowmovie.py

```diff
@@ -27,11 +27,10 @@
 # Defaults
 defaultIncrement = 4
 defaultDelay = 120
-defaultContrast = 1.0
```

Comment:
I figured because 1.0 is no change to contrast (the default) while the others are personal preference. But you're right, they're redundant now that there's a config file.